src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -24,7 +24,11 @@
 
  <div id="container">
    <h1>Register</h1>
-
+  
+   <% if(request.getAttribute("error") != null){ %>
+    <h2 style="color:red"><%= request.getAttribute("error") %></h2>
```

Comment:
Do you have a screenshot of what this looks like?

Reply:
Screenshot added! 👍

Reply:
:100: